projects/igniteui-angular/migrations/common/tsUtils.ts

```diff
@@ -214,6 +219,20 @@ export function createProjectService(serverHost: tss.server.ServerHost): tss.ser
     return projectService;
 }
 
+/**
+ * Attempts to get type definitions using the TypeScript Language Service.
+ * Can fall back to a cached version of the TSLS.
+ */
+function getTypeDefinitions(langServ: tss.LanguageService, entryPath: string, definition: tss.DefinitionInfo): any {
+    /*
+        getTypeScriptLanguageService is attached by us to the Typescript Language Service
+        via a custom made plugin, it's sole purpose is to cache the language service and return it
+        before any other plugins modify it
+    */
+    return langServ.getTypeDefinitionAtPosition(entryPath, definition.textSpan.start)
+        || (langServ as any).getTypeScriptLanguageService().getTypeDefinitionAtPosition(entryPath, definition.textSpan.start);
```

Comment:
You can probably use an extended interface for the service to maintain the type info here instead of loosing it with `any` :)